### ISSUE TITLE
Clarify formatting errors of history files upon validation

### ISSUE THREAD

**Issue by MattiSG** [OpenTermsArchive/engine#1055]
As seen in https://github.com/OpenTermsArchive/pga-declarations/pull/235 and https://github.com/OpenTermsArchive/pga-declarations/pull/236, when running validation on improperly formatted history files, the error message is impossible to understand:

```
 Uncaught error outside test suite:
     Uncaught TypeError: history[entry].sort is not a function
```

The validation should be clearer and yield a message similar to:

```
History file does not have the expected structure. Each term type should be declared within an array, rather than an object.
```

Similar to #832.

**Comment by gnaaruag** [comment]
hello there,
i'm relatively new to open source and think this is a good issue for me to get started. can i work on this/get this issue assigned?

**Comment by MattiSG** [comment]
Thanks @gnaaruag for offering your help! 😃 
While this issue is good for newcomers to the codebase, it probably is not easy enough to get started with as a first contribution to open source. I removed the label.
Could you tell us where you spotted this issue and what led you to this repository? 🙂

**Comment by gnaaruag** [comment]
i was browsing the github education's section on open source repos one can contribute to and i really found this to be interesting work. you do mention its probably not a good issue to get started with but can i still give it a shot?

**Comment by MattiSG** [comment]
Of course, you're welcome to try! :+1: However, we will unfortunately not be able at this time to provide proper guidance for a new contributor. You can find more information on https://github.com/OpenTermsArchive/engine/issues/832#issuecomment-1116069754.

Please also kindly note that we will not accept AI-generated pull requests.

**Comment by gnaaruag** [comment]
hey! thanks, ill spend a little while getting familial with the repo and then try to solve it. also do you guys have a discord server perchance?

**Comment by MattiSG** [comment]
We don't use Discord but have a Mattermost for contributors 🙂 Let's keep GitHub to discuss work on this issue so we have public traces!

**Comment by cybraia** [comment]
Hi @MattiSG is the issue still open? If yes, I would like to work on it

**Comment by MattiSG** [comment]
Hi @cybraia! Thank you for your interest 🙂 Please refer to https://github.com/OpenTermsArchive/engine/issues/1055#issuecomment-2101020120.
